ext/typerecipes.jl

```diff
@@ -4,7 +4,7 @@
 Makie.plottype(sol::AbstractODESolution, arg...; kw...) = Lines
 
 # prescribe the keyword arguments for the plot function
-Makie.used_attributes(::Makie.PlotFunc, ::AbstractODESolution, arg...; kw...) =
+Makie.used_attributes(::Type{<:Plot}, ::AbstractODESolution, ::Vararg{Any}) =

```

Comment:
Is this to improve compatibility?

Reply:
No. This will cause ambiguity error on Makie v0.20.0.